packages/timeline/src/ReactGantt.ts

```diff
@@ -345,19 +439,17 @@ export class ReactGantt extends SVGZoomWidget {
 
     }
 
-    private _maxBucket: number = 0;
-    private _minRangeWidth: number = Infinity;
-    calcBuckets(startKey: string | number, endKey: string | number) {
+    calcBuckets(data, startKey: string | number, endKey: string | number) {
```

Comment:
Mark as private and it will fix you unit test.  The issue is your returning a "d3 scaleLinear" publicly, but haven't declared "d3-scale" as a dependency - so end users will run into typing issues.

Reply:
Yep, that was it. This was passing tests locally (after disabling comms tests)